src/components/video/VideoPlayer.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+
+import VideoJS from './VideoJS';
+
+const hlsExtension = '.m3u8';
+const defaultOptions = {
+  autoplay: true,
```

Comment:
Why we autoplay the video? is this requirement?

Reply:
Yes, We initially hide the video-js player and only show it once user clicks a custom play button. So, by the time video-js loads user has already clicked the play button and we should autoplay the vide instead of asking user to click another play button.